Formula/phoronix-test-suite.rb

```diff
@@ -29,38 +30,38 @@ def install
  mkdir -p $DESTDIR$INSTALL_PREFIX/share/doc/phoronix-test-suite/
 -mkdir -p $DESTDIR$INSTALL_PREFIX/../etc/bash_completion.d/
 +mkdir -p $DESTDIR$INSTALL_PREFIX/etc/bash_completion.d/
- 
```

Comment:
Yes, this whole block is still needed to use the install script.
The other option would be to remove the bash-completion line and use `bash_completion.install` to do the completion install, but that would change the syntax more than the current patch.